### PR TITLE
[ACT-574] Added missing capabilities and identifiers

### PR DESCRIPTION
**Pull request by Szabadember**

### CODE REVIEW

**Review by lpusok — APPROVED** [review]

File: autoprovision/entitlements.go [hunk 1]
```diff
@@ -195,6 +195,16 @@ func (e Entitlement) Capability() (*appstoreconnect.BundleIDCapability, error) {
 			},
 		}
 		capSetts = append(capSetts, capSett)
+	} else if capType == appstoreconnect.SignInWithApple {
+		capSett := appstoreconnect.CapabilitySetting{
+			Key: appstoreconnect.AppleIDAuthAppConsent,
+			Options: []appstoreconnect.CapabilityOption{
+				appstoreconnect.CapabilityOption{
+					Key: "PRIMARY_APP_CONSENT",
```
Comment (godrei) [inline autoprovision/entitlements.go:203]: NOTE: this way we only support the most basic usage of SignInWithApple.
When you do not `Group with an existing primary App ID`
neither set `Server to Server Notification Endpoint`

Screenshot 2020-09-24 at 9 43 11

File: appstoreconnect/capabilities.go [hunk 2]
```diff
@@ -37,53 +37,68 @@ const (
...
+	SystemExtensionInstall         CapabilityType = "SYSTEM_EXTENSION_INSTALL"
+	UserManagement                 CapabilityType = "USER_MANAGEMENT"
+	SignInWithApple                CapabilityType = "APPLE_ID_AUTH"
+	ParentApplicationIdentifiers   CapabilityType = "ODIC_PARENT_BUNDLEID"
+	OnDemandInstallCapable         CapabilityType = "ON_DEMAND_INSTALL_CAPABLE"
 )
 
 // ServiceNameByKey maps iOS specific entitlement keys to names
 // based on the entitlements list: https://developer.apple.com/documentation/bundleresources/entitlements.
 var ServiceNameByKey = map[string]string{
```
Comment (godrei) [inline appstoreconnect/capabilities.go:51]: NOTE: seems to be unused

**Review by godrei — APPROVED** [review]

File: autoprovision/entitlements.go (hunk 1 above)
Comment (Szabadember) [reply autoprovision/entitlements.go:219]: Step will display a warning for this.

File: appstoreconnect/capabilities.go (hunk 2 above)
Comment (Szabadember) [reply appstoreconnect/capabilities.go]: Removed

**Review by lpusok — APPROVED** [review]

**Pull request merged**